dist/bin/upgrade-grist

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+docker compose pull
+
+if systemctl is-active --quiet grist.service; then
+  sudo systemctl restart grist.service
+
+  # Old docker images can quickly add up, so let's clean those up
+  docker system prune -a --force
```

Comment:
Any chance we prune images that they might not want us to prune with this? I.e, if they decide to run another service on the box.

Reply:
I looked into filters, and it complicated the situation too much for my taste, as there's a difference between images built by `docker compose pull` as opposed images that were merely pulled.

I am going to keep the current situation of using pruning indiscriminately, as it only removes unused resources.

Reply:
Can I suggest `docker image prune` and `docker builder prune` as an alternative then? As this would also take down any stopped containers, which might be off for other reasons (e.g maintenance)

Reply:
Okay, that seems reasonable.